### ISSUE TITLE
Send acceptance emails for article submissions

### ISSUE THREAD

**Issue by simholt** [osulp/Scholars-Archive#2536]
### Descriptive summary

Users get notification emails throughout the submission process. Users who upload to the Article worktype are not getting any notification that their deposit is live (articles do not go through a review process). I'm curious if this is leftover from ED2 stuff and just has to be turned back on? I tested this on staging and got nothing.

### Expected behavior
User deposits article and gets confirmation of their accepted submission along with the pURL.

### Actual behavior
No notifications; user has to go back to SA and look up their article to get the pURL.

### Steps to reproduce the behavior

1. Upload article.
2. Check out empty email and SA notifications page.

### Related work

 #708

**Comment by lamtu1** [comment]
Hi @simholt I just want to get clarification on this. So, when a user submit an `article`, they would then receive a confirmation email that their `article` is deposited and they can visit their public page via the `personal URL` where their page is located on SA. Is this correct?

**Comment by simholt** [comment]
Yes, like the email people get when their work is approved. I **thought** this was happening previously but it isn't now.

**Comment by lamtu1** [comment]
Yes I see that too and I will get to the implementation of it and thank you for clarifying it with me.

**Comment by simholt** [comment]
Alright -- the confirmation email came through to my email but I would prefer this work like other notifications and also goto the user's ScholarsArchive inbox like our regular submission notifications.

I have updated the message text so it matches our other notifications.

> SUBJECT LINE: 
> ScholarsArchive@OSU Message: Thank you for your deposit to ScholarsArchive@OSU
> 
> MESSAGE TEXT:
> ScholarsArchive@OSU has received your deposit of: [TITLE] 
> The citable URL for your article is: [URL]
> 
> If you have questions, please respond to this email.
> Thank you,
> ScholarsArchive@OSU Admin

**Comment by lamtu1** [comment]
Hi @simholt when you said it goes through the notification, meaning the bell icon next to your profile name when you log into the SA? Do you want both the email sent out to their email and the inbox? or just the inbox?

**Comment by simholt** [comment]
@lamtu1 Email and inbox!

**Comment by simholt** [comment]
This is not working -- encountered problems uploading articles -- see #2545

**Comment by lamtu1** [comment]
Hi @simholt , when you get a chance, can you give it a test again on staging? I just gave it a test on there once Corey fix the staging issue with creating/editing work on there, and when submit an article, I do see in the notification area is populated with the message. Let me know, if it doesn't appear on your end once you try it and I can go from there.

**Comment by simholt** [comment]
It worked! I got an email notification and one in my SA messages.

**Comment by carakey** [comment]
Looks like we reverted to old behavior on this issue. I just deposited an Article and got the standard email notification including: "Your item is under review by repository administrators. You will be notified if your deposit requires additional changes and/or when your deposit is live in the repository." (instead of the special Article deposit message).

**Comment by lamtu1** [comment]
`QA:` I just reload the `workflow` can I have this QA again creating a new `article` work and see if the message get deliver through?

**Comment by simholt** [comment]
QA pass -- looks good!
